storage/watchable_store_bench_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"math/rand"
+	"os"
+	"testing"
+)
+
+func BenchmarkWatchableStoreUnsyncedCancel(b *testing.B) {
+	watcherSize := b.N
+	s := &watchableStore{
+		KV:       newStore(tmpPath),
+		unsynced: make(map[*watcher]struct{}),
+		synced:   make(map[string][]*watcher),
```

Comment:
Need this to make the test not crash from assigning to nil map. This doesn't get populated though.